### ISSUE TITLE
undefined method `type' for 0:Sass::Script::Value::Number

### ISSUE THREAD

**Issue by maurizi** [brigade/scss-lint#45]
I received this error when trying to run `scss-lint` from a fresh install:

```
SCSSLint::Linter::Shorthand raised unexpected error: 'undefined method `type' for 0:Sass::Script::Value::Number'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter/shorthand.rb:38:in `check_script_string'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter/shorthand.rb:16:in `visit_prop'
/var/lib/gems/1.9.1/gems/sass-3.3.0.rc.1/lib/sass/tree/visitors/base.rb:37:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:61:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:68:in `block in visit_children'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:66:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:66:in `visit_children'
/var/lib/gems/1.9.1/gems/sass-3.3.0.rc.1/lib/sass/tree/visitors/base.rb:39:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:61:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:68:in `block in visit_children'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:66:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:66:in `visit_children'
/var/lib/gems/1.9.1/gems/sass-3.3.0.rc.1/lib/sass/tree/visitors/base.rb:39:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:61:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:68:in `block in visit_children'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:66:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:66:in `visit_children'
/var/lib/gems/1.9.1/gems/sass-3.3.0.rc.1/lib/sass/tree/visitors/base.rb:39:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:61:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:68:in `block in visit_children'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:66:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:66:in `visit_children'
/var/lib/gems/1.9.1/gems/sass-3.3.0.rc.1/lib/sass/tree/visitors/base.rb:39:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:61:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/linter.rb:14:in `run'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/runner.rb:49:in `block in find_lints'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/runner.rb:47:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/runner.rb:47:in `find_lints'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/runner.rb:30:in `block in run'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/runner.rb:29:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/runner.rb:29:in `run'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/lib/scss_lint/cli.rb:65:in `run'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/bin/scss-lint:7:in `block in <top (required)>'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/bin/scss-lint:5:in `tap'
/var/lib/gems/1.9.1/gems/scss-lint-0.12.0/bin/scss-lint:5:in `<top (required)>'
/usr/local/bin/scss-lint:23:in `load'
/usr/local/bin/scss-lint:23:in `<main>'
```

This was on Ubuntu 13.04 using `sudo gem install scss-lint`
